contrib/completion/bash/docker

```diff
@@ -334,14 +334,26 @@ _docker_diff() {
 
 _docker_events() {
 	case "$prev" in
-		--since)
+		--filter|-f)
+			COMPREPLY=( $( compgen -S = -W "container event image" -- "$cur" ) )
+			compopt -o nospace
+			return
+			;;
+		--since|--until)
+			return
+			;;
+	esac
+
+	case "${words[$cword-2]}$prev=" in
+		*event=*)
+			COMPREPLY=( $( compgen -W "create destroy die export kill pause restart start stop unpause" -- "${cur#=}" ) )
```

Comment:
@tianon @jfrazelle Well, this works (completion`--filter event=XXX`). Do you know any better way to realize subcompletions here? The problem here is that depending on context, I have to consider the _second last_ or _previous_ word. `$prev` might be "=".